lib/iex/lib/iex/server.ex

```diff
@@ -82,7 +82,8 @@ defmodule IEx.Server do
               {^ref, nil} ->
                 {:error, :refused}
               {^ref, leader} ->
-                IEx.Evaluator.start(server, leader)
+                {_, evaluator_state} = run_state(opts)
```

Comment:
This will also bring some benefits like: IEx.Evaluator.load_dot_iex no longer needs to be public. :)